tests/bluetooth/ctrl_llcp/src/main.c

```diff
@@ -1511,25 +1511,38 @@ void test_api_remote_encryption_start_no_ltk(void)
 
 void test_main(void)
 {
-	ztest_test_suite(test,
-			 ztest_unit_test(test_api_init),
+	ztest_test_suite(internal,
 			 ztest_unit_test(test_int_mem_proc_ctx),
 			 ztest_unit_test(test_int_mem_tx),
 			 ztest_unit_test(test_int_mem_ntf),
 			 ztest_unit_test(test_int_create_proc),
-			 ztest_unit_test(test_int_pending_requests),
+			 ztest_unit_test(test_int_pending_requests)
+			);
+
+	ztest_test_suite(public,
+			 ztest_unit_test(test_api_init),
 			 ztest_unit_test(test_api_connect),
-			 ztest_unit_test(test_api_disconnect),
+			 ztest_unit_test(test_api_disconnect)
+			);
+
+	ztest_test_suite(version_exchange,
```

Comment:
Eventually we should move these tests to their own test-suite/module/files

Reply:
Agree